ldms/man/ldmsd_controller.man

```diff
@@ -787,6 +776,47 @@ The base value of user data (uint64)
 Increment value (int). The default is 0. If incr is 0,
 the user data of all matched metrics are set
 to the base value. Optional.
+.RE
+
+.SS Change the security parameters of LDMS sets using regular expression.
+The command does not affect remote sets, i.e., sets aggregated from a remote
+LDMSD. The new security parameter values will only affect new clients or
+connections. That is, the aggregators that already have access to the sets will
+not lose access.
```

Comment:
@nichamon I think we discussed doing a set_delete followed by reconstituting the set with the new permissions. Have we looked into that?

Reply:
@tom95858 I looked into it but forgot to continue the discussion. A possible race exists between LDMS deleting the set and the plugin updating the metric values. 

I looked into unpublishing the set, changing the permission, and republishing the set. The sampler daemon will not free the set memory, but the clients will go through a process similar to when receiving a SET_DELETE request. This approach prevents the race I mentioned above. The current ldms_set_unpublish() implementation does not support what I want to do. It does not send a dir_del request to clients.